src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -70,6 +57,35 @@ function TypeOfCarePage({
     }
   }, []);
 
+  const { data, schema, setData, uiSchema } = useFormState({
+    initialSchema: () => {
```

Comment:
This logic was in the reducer before, but now it's inline with the component and you don't have to jump between files (or even back up to the schema variables) to see what's happening.